contrib/test_jenkins.sh

```diff
@@ -582,38 +589,38 @@ start_daemon() {
 #
 # Run UCX performance daemon test
 #
-run_ucx_perftest_daemon() {
+run_ucx_perftest_with_daemon() {
 	ucx_inst_ptest=$ucx_inst/share/ucx/perftest
 
 	ucx_perftest="$ucx_inst/bin/ucx_perftest"
 	ucx_perftest_daemon="$ucx_inst/bin/ucx_perftest_daemon"
 	ucp_test_args="-b $ucx_inst_ptest/test_types_ucp_daemon"
 
-	devices="$(get_active_ib_devices)"
-	devices="$(get_ib_bf_devices $devices)"
-
+	devices="$(get_ib_bf_devices $(get_active_ib_devices))"
 	my_devices=$(get_my_tasks $devices)
 	for ucx_dev in $my_devices
 	do
-		echo "==== Running ucx_perftest_daemon kit on $ucx_dev ===="
-		export UCX_NET_DEVICES=$ucx_dev,lo
+		echo "==== Running ucx_perftest over a daemon on $ucx_dev ===="
+		ip_addr=$(get_rdma_device_ip_addr $ucx_dev)
+		if [ -z "$ip_addr" ]
+		then
+			echo "Cannot find IPv4 address for device $ucx_dev"
+			continue
+		fi
 
-		# We explicitly disable cuda transport, because it's not p2p and therefore
-		# imposes INVALIDATE_RMA flag for all lanes (@see ucp_ep_config_init).
-		# However invalidating of imported rkeys is not supported by the daemon.
-		# TODO: Should we support invalidation of imported keys?
-		# Normally cuda_ipc cannot be used to communicate between host and DPU,
-		# unless we run both processes on host for testing purposes.
-		export UCX_TLS=^cuda
+		export UCX_NET_DEVICES=$ucx_dev
+		export UCX_MAX_RNDV_LANES=1
```

Comment:
why needed?

Reply:
It works without this option set, removing